migrations/30_octobay.js

```diff
@@ -37,7 +39,10 @@ module.exports = function (deployer, network) {
       })
       OctobayGovernor.deployed(OctobayGovernorInstance => {
         OctobayGovernorInstance.setOctobay(octobayInstance.address)
-      })      
+      })     
+      OctobayGovNFT.deployed(OctobayGovNFTInstance => {
```

Comment:
This needs to be `deployed().then(...`. Already changed and pushed this for the other contracts in this file.